src/nmdc_submission_schema/schema/nmdc_submission_schema_base.yaml

```diff
@@ -69,6 +69,11 @@ enums:
       Pellet:
       TE:
       Water:
+  JGIMatCatEnum:
```

Comment:
Should the word "Biosafety" be in the enum name to make its usage more clear? Something like `JGIBiosafetyMaterialCategoryEnum`? (We don't get charged by the character; we can use as many as we want)

Reply:
```suggestion
  JGIBiosafetyMaterialCategoryEnum:
```